identify/README.md

```diff
@@ -83,6 +86,13 @@ message Identify {
 }
 ```
 
+### sequence
+
+A sequence number for this identify message used to deal with reordered Identify
+messages. Senders SHOULD increment the sequence number for every Identify message
+they send. After processing an Identify with a certain sequence number, receivers
+MUST ignore all Identify messages with smaller sequence numbers.
```

Comment:
I am assuming that the monotonic sequence number is global, i.e. not per peer nor per connection, correct?

What happens on restarts or crashes? I am assuming that nodes are not expected to persist their sequence numbers, correct?

A timestamp instead of a simple counter might be an option. Though what happens if the node drifts in time? One could require that the timestamp is monotonically increasing, i.e. that time adjustments are only ever allowed to happen in forward direction.

Reply:
We can reuse this https://github.com/libp2p/specs/blame/master/RFC/0003-routing-records.md#L115-L119

Reply:
Yeah, I was planning to use a (monotonic) clock here, with some extra protections to make sure that we don't send the same sequence number twice. This will break if we send more than one Identify message per nanosecond, but I think we can live with that :)

Reply:
> We can reuse this https://github.com/libp2p/specs/blame/master/RFC/0003-routing-records.md#L115-L119

Good idea @Menduist. I suggest adopting that paragraph verbatim here.

Reply:
Added in d9de900.